src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random;
+    private final ColorSupplier supplier;
+
+    public Lottery(Random random, ColorSupplier supplier) {
+        this.random = random;
+        this.supplier = supplier;
+    }
+
+    public Ball getRandomBall() {
+        int limit = 100;

```

Comment:
`limit` should be consntant, not local variable. Remember, constant always `static final`, also put `public` modifier.  Also, constants are UPPER case and SNAKE case.